src/Collector/ValueObject/CollectedTemplateRender.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Efabrica\PHPStanLatte\Collector\ValueObject;
+
+use Efabrica\PHPStanLatte\Template\Variable;
+use PHPStan\PhpDoc\TypeStringResolver;
+use PHPStan\ShouldNotHappenException;
+
+/**
+ * @phpstan-type CollectedTemplateRenderArray array{templatePath: string|bool|null, variables: array<array{variableName: string, variableType: string}>, className: string, methodName: string, file: string, line: int}
+ */
+final class CollectedTemplateRender
```

Comment:
This class representr render call to template. Either from render() call in PHP or from including tempalte in latte.